lib/display.rb

```diff
@@ -1,5 +1,21 @@
 class Display
+  def initialize(console:)
+    @console = console
+  end
+
   def present(board)
+    console.output(board_display(board))
+  end
+
+  def message(message)
+    console.output(message)
```

Comment:
That's a good question. I wanted to keep it so that Console has a single responsibility of general input/output. I'm not sure how it would affect that responsibility if I added the job of knowing the desired board format. Are those two things closely related enough to be considered the same?

Reply:
I see what you mean, but I feel that having them separated in two classes require too many extra steps.

Further down the road, if it starts bloating with new requirements, we can always refactor it in a useful abstraction.